deps/v8/src/api.cc

```diff
@@ -6663,6 +6663,11 @@ Local<String> v8::String::Concat(Local<String> left, Local<String> right) {
   return Utils::ToLocal(result);
 }
 
+Local<String> v8::String::Concat(Local<String> left, Local<String> right) {
+  i::Handle<i::String> left_string = Utils::OpenHandle(*left);
```

Comment:
A bit hacky (opening a handle just to fetch the isolate), but idk if there's a better way. @addaleax do you?

Perhaps one could get the current `env` and extract the associated `isolate`? Idk how different the result would be, but it's certainly less hacky IMO.

Reply:
That's just copied from the V8 source code (see the Refs link)

Reply:
I get it. It works so it shouldn't be a problem anyway.